### DISCUSSION TITLE
Set custom error unrelated to form fields, that doesn't prevent form submit

### DISCUSSION THREAD

**Discussion by arturmolenda** [react-hook-form/react-hook-form#6276] [Q&A] [answered]
Hello,

I'm wondering what would be the correct way to show errors that you can't pin under any of form fields, errors like **_Something went wrong, try again_** 

```
  setError("customError", {
            type: "manual",
            message: "Error message",
          });
``` 
seems like a way to go but the only problem I'm having is that, this error prevents my form submit, anyone knows proper way to handle this?

**Comment by bluebill1049** [comment, marked as answer]
custom error will required a manul clearError, if that's your problem.

**Reply by arturmolenda** [reply]
got it, thanks for answering
